buildenv/jenkins/Jenkinsfile

```diff
@@ -73,19 +73,16 @@ pipeline {
         }
     }
     post {
-    	failure {
```

Comment:
I think you removed this while you were trying things out, but don't accidentally remove the slack notifications piece from the jenkinsfile

Reply:
Yes, I'll add this back before delivering. I had removed it just to turn off the notification while I am running this job off of my branch.

Reply:
This has been put back.